process/block/preprocess/transactions.go

```diff
@@ -906,11 +906,21 @@ func (txs *transactions) getAllTxsFromMiniBlock(
 }
 
 func (txs *transactions) getRemainingGasPerBlock() uint64 {
-	gasConsumed := txs.getTotalGasConsumed()
+	gasProvided := txs.getTotalGasConsumed()
```

Comment:
Actually this is really consumed as in getTotalGasConsumed method, from total gas provided is subtracted gas penalized and gas refunded if they exist

Reply:
done